panoptes_client/subject_set.py

```diff
@@ -34,4 +34,15 @@ def add_subjects(self, subjects):
             json={'subjects': _subjects}
         )
 
+    def add_subject_link(self, subject_id):
```

Comment:
`add_subjects` already allows adding a single subject (see lines 23-24). If you also want to be able to pass a subject ID as well as a `Subject` instance, the best thing to do would be to add a type check inside the for loop at line 27 to see if `subject` is an int, string, or unicode.